manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Astronomer.Astro
+PackageVersion: 1.17.0
+InstallerLocale: en-US
+InstallerType: portable
```

Comment:
```suggestion
InstallerType: nullsoft
```

Reply:
Why would we need a `nullsoft` installer type? This is a standalone exe file without any installer.